dev/src/overmind/actions.tsx

```diff
@@ -143,8 +143,15 @@ export const updateSubmission = async ({ state, effects }: Context, status: Subm
         return
     }
 
+    if (state.activeSubmissionLink?.getAssignment()?.getIsgrouplab()) {
+        actions.loopThroughAndUpdate({ links: state.courseGroupSubmissions[state.activeCourse], status: status })
+    }
+    actions.loopThroughAndUpdate({ links: state.courseSubmissions[state.activeCourse], status: status })
```

Comment:
Should this be called both inside the for loop and here? That is, should it be called twice if the if-condition is satisfied?

Reply:
Currently `state.GroupSubmissions` and `state.courseSubmissions` are two separate lists. 

I added a button to switch between group view and student view in review and results. Where group view lists submissions by group, while student view lists submissions by user. 

When updating a group submission I want both lists to reflect the updated status. This way, when you update status of a submission in student view, the submission will also have the updated status if you switch to group view.

Reply:
I now see they aren't identical. Resolving.